### PR TITLE
Add container memory limit parameter

### PR DESCRIPTION
**Pull request by volodymyr-mykhailyk**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@
     image: '{{ postgres_docker_image }}:{{ postgres_docker_image_tag }}'
     restart_policy: always
     state: started
+    memory: '{{ container_memory_limit }}'
```
Comment (Kukunin) [inline tasks/main.yml]: can you make it nil by default with ability to overwrite?

Comment (volodymyr-mykhailyk) [reply tasks/main.yml]: Why not limit by default?

Comment (Kukunin) [reply tasks/main.yml]: http://docs.ansible.com/ansible/playbooks_filters.html#omitting-parameters

**Comment by volodymyr-mykhailyk** [comment]
@Kukunin Updated default to nil value

**Pull request merged**